purchase_product_usage/views/purchase_order_line_view.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase.order.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_line']/tree/field[@name='name']"
+                   position="after">
+                <field name="usage_id"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="purchase_order_form_form" model="ir.ui.view">
```

Comment:
why are there two records for the same inherited view (purchase.purchase_order_form)?
 <record id="purchase_order_form"
and
<record id="purchase_order_form_form"

Reply:
solved